examples/00-EDB/15_ac_analysis.py

```diff
@@ -0,0 +1,131 @@
+"""
+EDB: SYZ analysis
+-------------------
+This example shows how you can use PyAEDT to set up SYZ analysis on Serdes channel.
+The input is the name of the differential nets. The positive net is PCIe_Gen4_TX3_CAP_P.
+The negative net is PCIe_Gen4_TX3_CAP_N. The code will place ports on driver and
+receiver components.
+"""
+
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes importing a section.
+
+import time
+import pyaedt
+
+###############################################################################
+# Download file
+# ~~~~~~~~~~~~~
+# Download the AEDB file and copy it in the temporary folder.
+
+temp_folder = pyaedt.generate_unique_folder_name()
+targetfile = pyaedt.downloads.download_file('edb/ANSYS-HSD_V1.aedb', destination=temp_folder)
+time.sleep(5)
+
+print(targetfile)
+
+###############################################################################
+# Configure EDB
+# ~~~~~~~~~~~~~
+# Launch the :class:`pyaedt.Edb` class, using EDB 2024 R1.
+
+edbapp = pyaedt.Edb(edbpath=targetfile, edbversion="2024.1")
```

Comment:
example must be running 2023.2 since 2024.1 is not yet installed
```suggestion
edbapp = pyaedt.Edb(edbpath=targetfile, edbversion="2023.2")
```

Reply:
LGTM